packages/story-editor/src/types/pageDataUrls.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import type { Page } from '@googleforcreators/elements';
+
+export interface PageDataUrlsState {
+  dataUrls: Record<string, string>;
+}
+
+export interface PageDataUrlsActions {
+  queuePageImageGeneration: (Page: Page) => void;
+}
+
+export interface PageDataUrlsContext {
+  state: PageDataUrlsState;
+  actions: PageDataUrlsActions;
+}
```

Comment:
So this would be:

```ts
export type PageDataUrls = Record<string, string>;
export type QueuePageImageGeneration: (page: Page) => void;
export interface PageDataUrlsContext {
  state: { dataUrls: PageDataUrls }
  actions: { queuePageImageGeneration: QueuePageImageGeneration};
}
```